src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -190,6 +199,31 @@
  *         the very same way as it is done by HTTPS. If "false" the host name is not verified at all. Per default the
  *         hostname is verified (true).
  *     </li>
+ *
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.token</b><br>
```

Comment:
It is for provisioning endpoints, so I. would prefer the key `ch.hevs.cloudio.endpoint.provisioningToken`